docs/management/rollups/create_and_manage_rollups.asciidoc

```diff
@@ -129,12 +129,28 @@ rollup index, or you can remove or archive it using <<creating-index-lifecycle-p
 Your next step is to visualize your rolled up data in a vertical bar chart.
 Most visualizations support rolled up data, with the exception of Timelion, TSVB, and Vega visualizations.
```

Comment:
It's not related to this change, but I just noticed it says here TSVB doesn't support rolled up data - that's not true, you can query rolled up index patterns as well. I can create a separate issue for it or we can just fix it together with the rest.

Reply:
@flash1293 as the support for TSVB began in 7.2, we'll fix this in a separate PR.

Reply:
@flash1293 I added this change. @gchaps will keep an eye on the backports to 7.0 & 7.1.